cg/models/scout/scout_load_config.py

```diff
@@ -69,6 +69,13 @@ class ScoutBalsamicIndividual(ScoutIndividual):
     vcf2cytosure: Optional[str] = None
 
 
+class ScoutRnafusionIndividual(ScoutIndividual):
```

Comment:
hmmm... It's still a bit in the air which fields will be relevant to include. They might diverge so I'm not sure if I'm very keen to inherit from  ´ScoutBalsamicIndividual´. Also then we would also inherit vcf2cytosure which we don't need. But if you feel strong about this, I could change it for now to avoid redundancy and we can revisit it in the future.

Reply:
I agree with Vincent. `vcf2cytosure` is an optional field in ScoutBalsamicIndividual, so it would be a good idea to inherit it from Balsamic or create a new individual (ScoutCancerIndividual?)

Reply:
Made a ScoutCancerIndividual from which balsamic and rnafusion inherit.